### ISSUE TITLE
Export mods from collections to spreadsheet

### ISSUE THREAD

**Issue by quetschke** [DestinyItemManager/DIM#6247]
**Proposed change**
Export the mods from collections into a spreadsheet with the following columns (examples in [ .. ]:
Type: [Armor,Weapons,Equipment]
Subtype: [Arms,Chest, .. ,Legs, General, Ghost]
Name: [Anti-Barrier Pulse Rifle, ..]
Season: [Season 12, ..]
Energy: [1, ..]
Energytype: [Arc, Solar, Void, Any, ..] 
Source: [Seasonal Artefact, ..] 
Desc: [Grants weapon [Shield-Piercing] shield-piercing ..]
In Inventory: [Yes, No] (Probably means available on character)
In Collection: [Yes, No]  

The export interface could be added to Settings->Spreadsheets

The data is available in DIM, but some entries only become available after clicking or hovering over the mod.

No other bungie API access program/website provides the information in text/spreadsheet format.

**How does this fit into your workflow?**
Having a complete list of available mods, especially for armor, would make planning of loadouts easier. It is very useful to have all the information available in one place.

**Comment by bhollis** [comment]
How would you leverage the CSV format data? If you're writing scripts or complicated spreadsheets my suggestion would be to go to the API yourself, that way you'll always get what you want.

**Comment by quetschke** [comment]
I plan to use this list to create a spreadsheet with drop-down selections for a full mod load-out for armor.

I already followed your advice and was going to post the dictionary entries (using python) that I used to piece together the columns. [armor_mods_spreadsheet_for_beyond_light](https://www.reddit.com/r/DestinyTheGame/comments/k474gf/armor_mods_spreadsheet_for_beyond_light/?utm_source=share&utm_medium=web2x&context=3)

I wasn't aware that the info was spread out between collection, item and perks entries and the "available to character" info is still missing. I have not looked into that, but I assume this involves comparing the list with the character inventory.
